app/components/channel_list/channel_list.tsx

```diff
@@ -38,13 +26,13 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
 }));
 
 type ChannelListProps = {
-    currentTeamId?: string;
     iconPad?: boolean;
     isTablet: boolean;
     teamsCount: number;
+    currentTeamId: string;
```

Comment:
this must be optional and handled accordingly.. display an error screen that teams could not be loaded as in the code you deleted.